src/cdk/stepper/stepper-button.ts

```diff
@@ -12,7 +12,10 @@ import {CdkStepper} from './stepper';
 /** Button that moves to the next step in a stepper workflow. */
 @Directive({
   selector: 'button[cdkStepperNext]',
-  host: {'(click)': '_stepper.next()'}
+  host: {
+    '(click)': '_stepper.next()',
+    'type': 'button'
```

Comment:
I removed `'type': 'button'`. I'll add it to the docs instead as @kara suggested.